schemas/org.gnome.shell.extensions.org-scrollwm.gschema.xml

```diff
@@ -371,6 +371,11 @@
         <summary>Duration of animations in seconds</summary>
     </key>
 
+    <key type="b" name="found-dash-to-panel">
+        <default>false</default>
+        <summary>Automatically set on startup when Dash to Panel is enabled. Any changes to this setting will be reset.</summary>
+    </key>
+
```

Comment:
Apparently `prefs.js` can't access extension constants -- it exists outside of the extension manager, so I had the extension leave a message for the prefs view to find afterwards.